src/content/reference/react-dom/components/option.md

```diff
@@ -4,12 +4,13 @@ title: "<option>"
 
 <Intro>
 
-The [built-in browser `<option>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) lets you render an option inside a [`<select>`](/reference/react-dom/components/select) box.
+[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) memungkinkan Anda untuk merender opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
```

Comment:
```suggestion
[Komponen `<option>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) memungkinkan Anda untuk me-*render* opsi di dalam kotak [`<select>`](/reference/react-dom/components/select).
```

Italicise the word *render* as per the [Glossary](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium).